README.md

```diff
@@ -1,11 +1,85 @@
 [![Build Status](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client.svg?branch=master)](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client)
 
-pint
-====
+# Introduction
 
 pint (Public Cloud INformation Tracker) is a command line client to access
 the REST API provided by the SUSE Public Cloud Information Service. The
 information service provides data about the images SUSE publishes in various
 public cloud frameworks (Amazon, Google, Microsoft) as well as the
 servers that make up the SUSE maintained and operated update infrastructure
 in those frameworks.
+
+## Quick Start
+
+There are various ways to install the Pint Client:
+
+### Install On SUSE Linux using rpm package  
+
+```bash
+  $ sudo zypper in python3-susepubliccloudinfo
+```
+
```

Comment:
This probably warrants a comment about the distros and repositories. For example SLES is "SUSE Linux" and that command will fail in a generic install as the Public Cloud Module will be missing.